docs/podman-replay.1.md

```diff
@@ -0,0 +1,20 @@
+% podman-replay(1)
+
+## NAME
+podman\-container - replay pods and containers based on a structured input file
+
+## SYNOPSIS
+**podman replay** *subcommand*
+
+## DESCRIPTION
+The replay command will recreate pods and containers based on the input from a structured (like YAML)
+file input.  Containers will be automatically started.
+
+## COMMANDS
+
+| Command  | Man Page                                            | Description                                                                  |
+| -------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
+| kube | [podman-replay-kube(1)](podman-replay-kube.1.md)              | Recreate pods and containers based on Kubernetes YAML.
+
+## SEE ALSO
+podman, podman-pod, podman-container, podman-generate
```

Comment:
Ditto (1) here

Reply:
Probably wouldn't hurt to add podman-replay-kube(1) here and 'podman-replay(1) to the prior.

Reply:
done